javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  var array = [];
+  for (var i = 0; i < values.length; i++) {
+    if (values[i] === null) {
+      values.splice(i, 0);
```

Comment:
Oh no, this is changing the original array. If the current value is null, just ignore it, don't delete it from the argument array